ietf-optical-impairment-topology.tree

```diff
@@ -28,15 +29,21 @@ module: ietf-optical-impairment-topology
                 |  +--ro amplifier
                 |     +--ro type-variety    string
                 |     +--ro operational
-                |        +--ro actual-gain                             decimal64
-                |        +--ro tilt-target                             decimal64
-                |        +--ro out-voa                                 decimal64
-                |        +--ro in-voa                                  decimal64
+                |        +--ro actual-gain
+                |        |       decimal64
+                |        +--ro tilt-target
```

Comment:
do we have a crisp definition at which point in frequency the tilt/gain pair applies? i.e. need to define a point and a slope. If it can't be defined in the description, another parameter would need to be added.

Reply:
@ggrammel : Hi Gert, I've open a specific issue for your comment.